jormungandr/src/stats_counter.rs

```diff
@@ -16,6 +17,7 @@ struct StatsCounterImpl {
     block_recv_cnt: AtomicUsize,
     start_time: Instant,
     slot_start_time: AtomicU64,
+    tip_block: RwLock<Option<Block>>,
```

Comment:
I don't know for sure if this should be a `tokio::RWLock` instead. It felt confusing the mix we have right now with the futures implementations versions plus the tokio version itself.

Reply:
I think there's a way to make this lockless, some kind of Cell+Arc ?

Reply:
Well, since it needs to change and I saw that the other `StatsCounterImpl ` attributes are atomic I decided to use that. But we can use `Cell` if it is only accessed synchronously.